ci/deps/azure-37-locale.yaml

```diff
@@ -3,8 +3,18 @@ channels:
   - defaults
   - conda-forge
 dependencies:
-  - beautifulsoup4
+  - python=3.7.*
+
+  # tools
   - cython>=0.29.13
+  - pytest>=4.0.2
+  - pytest-xdist>=1.21
```

Comment:
actually looking at some of our builds, I think we need pytest-xdist>1.29, which likely implies pytest>5.0.1, if you can confirm this, we can move the pytest pin.

I think that we are likely installing the latest pytest which is fine; these are just a min pin. So on the 36-min versions, can you pin exactly these versions to test.

Reply:
in all our builds we have `pytest-xdist>=1.21`, happy to force `1.29` and `pytest>5.0.1` but I don't think it's needed.

I agree with your previous comments. I think we do care about the versions. My point was that I don't think we care about supporting old versions, and we should probably just use the latest. Probably not for black. It's already quite annoying to use it, if at every new version it reformats the code, we should probably pin it and update it once per year.